simple/src/main/java/org/example/EcomNewPaymentMain.java

```diff
@@ -0,0 +1,104 @@
+package org.example;
+
+import com.kodypay.grpc.pay.v1.KodyPayTerminalServiceGrpc;
+import com.kodypay.grpc.pay.v1.TerminalsRequest;
+import com.kodypay.grpc.pay.v1.TerminalsResponse;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.Metadata;
+import io.grpc.stub.MetadataUtils;
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.util.Properties;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+
+public class EcomNewPaymentMain {
+    private static final Logger LOG = LoggerFactory.getLogger(EcomNewPaymentMain2.class);
+    private static final long TIMEOUT_MS = java.time.Duration.ofMinutes(3).toMillis();
+    public static void main(String[] args) {
+
+        // Load configuration properties
+        Properties properties = loadProperties();
+        var address = properties.getProperty("address", "grpc-developement.kodypay.com");
```

Comment:
We don't want to expose externally development host. Use staging.
```suggestion
        var address = properties.getProperty("address", "grpc-staging.kodypay.com");
```